samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/ConfirmRemovePublishedAssessmentListener.java

```diff
@@ -79,10 +79,16 @@ public void processAction(ActionEvent ae) throws AbortProcessingException
           context.addMessage(null,new FacesMessage(err));
   		  author.setOutcome("author");
   		  return;
-        }
-
-    	author.setOutcome("confirmRemovePublishedAssessment");
-    	
+        }
+
+    	//Alert user to remove submissions associated with the assessment before delete the assessment bugid:4456 -Qu 6/23/2011
+        int submissions = publishedAssessmentService.getTotalSubmissionForEachAssessment(publishedAssessmentId);
+    	if (submissions > 0)

```

Comment:
Sakai adhere's to Sun coding standards, and as such braces are required for one line if/else statements. It may look pretty without the braces, but it can lead to problems down the road.